doc-src/source/data-structure.rst

```diff
@@ -112,6 +113,135 @@ use the `rotate2` method::
     + signal                   : + DATA GROUP
     + sys                      : + DATA GROUP
 
+Orientation Data
+................
+  
+The instrument orientation data in |dlfn| data objects is contained in
+the ``orient`` data group. The ``orientmat`` data item in this group
+is the orientation matrix, :math:`R`, of the instrument in the earth
+reference frame. It is a 3x3xNt array, where each 3x3 array is the `rotation matrix
+<http://en.wikipedia.org/wiki/Rotation_matrix>`_ that rotates vectors
+in the earth frame, $v_e$, into the instrument coordinate system,
+$v_i$, at each timestep:
+
+.. math:: v_i = R \cdot v_e
+
+The ENU definitions of coordinate systems means that the rows of
+:math:`R` are the unit-vectors of the XYZ coordinate system in the ENU
+reference frame, and the columns are the unit vectors of the ENU
+coordinate system in the XYZ reference frame. That is, for this kind
+of simple rotation matrix between two orthogonal coordinate systems,
+the inverse rotation matrix is simply the transpose:
+
+.. math:: v_e = R^T \cdot v_i
+
```

Comment:
Somewhere, add info about which instruments output an orientation matrix as part of the raw data, and which will have an orientation matrix built from the raw h,p,r data?